raiden/tests/conftest.py

```diff
@@ -68,40 +67,6 @@ def pytest_addoption(parser):
     )
 
 
-def load_fixtures(module_name: str) -> Dict:
-    """Load all fixture functions from a module"""
-    import importlib
-    module = importlib.import_module(module_name)
-    return {
-        k: v for k, v in module.__dict__.items()
-        if hasattr(v, '_pytestfixturefunction')
-    }
-
-
-def load_fixtures_list(module_list: str) -> Dict:
-    fixtures_all = {}
-    for module in module_list:
-        fixtures_all.update(load_fixtures(module))
-    return fixtures_all
-
-
-@pytest.hookspec()
-def pytest_configure(config):
```

Comment:
this required global state changes + fixtures of a separate repo, I removed this to streamline the usage of the fixtures (they already have many hops, and having to search for fixtures on multiple repos is not the best approach)